doc/classes/Array.xml

```diff
@@ -553,7 +549,7 @@
 				        return false
 
 				var my_items = [[5, "Potato"], [9, "Rice"], [4, "Tomato"]]
-				my_items.sort_custom(MyCustomSorter, "sort_ascending")
+				my_items.sort_custom(Callable(MyCustomSorter, "sort_ascending"))
```

Comment:
Isn't this the whole point of Callables?
```suggestion
				my_items.sort_custom(MyCustomSorter.sort_ascending)
```

Reply:
This won't work because of #41919, I already mentioned it above. Should the example be written regardless of this bug?

Reply:
Ah I had missed your comment, I guess if there's a bug you were right to do it this way.